var/spack/repos/builtin/packages/ace/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os import environ as env
+from spack import *
+
+
+class Ace(Package):
```

Comment:
I would switch this to `MakefilePackage`, rename `install` to `edit`, and drop the `make(), make('install')` lines.